packages/compass-components/src/components/editor.tsx

```diff
@@ -13,6 +13,8 @@ if (typeof window === 'undefined' && typeof globalThis !== 'undefined') {
 import 'ace-builds';
 import type { IAceEditorProps, IAceOptions } from 'react-ace';
 import AceEditor from 'react-ace';
+// TODO(COMPASS-6117): Re-enable so that we can use ace workers
+// import 'ace-builds/webpack-resolver';
```

Comment:
This seems to work when just uncommented, but I am worries that there might me some side-effects on tests that I don't want to deal with right now as this was just an attempt to clarify ace-mode code